up-transport-vsomeip/src/message_conversions.rs

```diff
@@ -269,6 +259,21 @@ where {
 
         Ok(vsomeip_msg)
     }
+
+    fn ucode_to_vsomeip_err_code(ucode: UCode) -> vsomeip::return_code_e {
+        match ucode {
+            UCode::OK => vsomeip::return_code_e::E_OK,
+            UCode::INVALID_ARGUMENT => vsomeip::return_code_e::E_WRONG_MESSAGE_TYPE,  
```

Comment:
@stevenhartley -- so mapping over from UCode to SOME/IP Error Code for those items in [the table](https://github.com/eclipse-uprotocol/up-transport-zenoh-rust/pull/72) there are cases where there are two possible SOME/IP Error Codes.

Which one should be chosen when mapping UCode => SOME/IP Error Code?

Reply:
This shouldn't hold up the PR, but it might be good if you add a // TODO here and then open an issue on the up-spec repo to clarify intention.

Reply:
food for thought: If the enums were bitvectors, we could use "OR" and encapsulate all the edge cases.

Reply:
Interesting thought, done some of these in my day. How would this work with one side being a Protobuf enum and the other being a C enum that we have to map back down into what vsomeip is expecting?

Reply:
AFAIK protobuf enums support non-sequential, and reserved enumerated values. Effectively you have to be able to change both sides for it to work correctly. VSomeIP mapping is basically 1:1, UCode mapping can be 1:2. 

pseudo eg:
``` 
// vsomeip values are bit unique
enum vsomeip_e { 
   E_OK = 0,
   E_WRONG_MESSAGE_TYPE  = 0x01,
   E_UNKNOWN_METHOD = 0x02,
   // NOTE value 0x03 is not allowed
   E_TIMEOUT = 0x04
   ...
};

enum UCode {
   OK = E_OK, // 00
   WRONG_MSG_TYPE = E_WRONG_MESSAGE_TYPE, // 0x01
   UNKNOWN_METHOD=   E_UNKNOWN_METHOD,  // 0x02
   INVALID_ARGUMENT  = ( WRONG_MSG_TYPE | UNKNOWN_METHOD) // 0x03
};

```

Reply:
I see where you're coming from. Let's leave as-is for now in this PR.